package exception

import "errors"

var ErrRoomExists = errors.New("room already exists")
